test/test_storage.cpp

```diff
@@ -1248,11 +1249,11 @@ TORRENT_TEST(readwritev_zero_size_files)
 	fs.add_file(combine_path("readwritev", "4"), 0);
 	fs.add_file(combine_path("readwritev", "5"), 6561);
 	fs.set_piece_length(0x1000);
-	fs.set_num_pieces((fs.total_size() + 0xfff) / 0x1000);
+	fs.set_num_pieces((int(fs.total_size()) + 0xfff) / 0x1000);
```

Comment:
I suppose this is still just part of the test, but the narrowing should happen after the division